examples/example.py

```diff
@@ -39,20 +40,17 @@
     scale_power_statistics = power_scaler(x=imputer_power_statistics)
 
     # Create lagged time series to later be used as regressors
-    shift_power_statistics = ClockShift(lag=1, name="ClockShift_Lag1"
-                                        )(x=scale_power_statistics)
-    shift_power_statistics2 = ClockShift(lag=2, name="ClockShift_Lag2"
-                                         )(x=scale_power_statistics)
+    lag_features = Select(start=-2, stop=0, step=1)(x=scale_power_statistics)
```

Comment:
Maybe we add more than tha last two values as lag features when forecasting 24 steps ahead

Reply:
Yes you are right. However, I would prefer to  do this in a separate PR together with the extension of the examples. I just replicated the old behaviour with the Select Module.
In the extended examples there should be more assertions etc. for using them also as integration tests.